index.bs

```diff
@@ -259,6 +261,38 @@ the user agent SHOULD run these steps:
 2. If |video| is {{pictureInPictureElement}}, run the <a>exit
     Picture-in-Picture algorithm</a>.
 
+## Auto Picture-in-Picture ## {#auto-pip}
+
+Some pages may want to automatically enter and leave Picture-in-Picture for
+a video element; for example, video meetings web apps would benefit from
+some automatic Picture-in-Picture behavior when user switches back and forth
+between web app and other applications/tabs.
+To support these use cases, a new {{autoPictureInPicture}} attribute is
+added to the list of content attributes for video elements.
+
+A corresponding {{autoPictureInPicture}} IDL attribute which reflects the
+value of element’s {{autoPictureInPicture}} content attribute is added to
+the {{HTMLVideoElement}} interface. The {{autoPictureInPicture}} IDL attribute
+MUST <a>reflect</a> the content attribute of the same name.
```

Comment:
Sounds like the two sentences say the same thing, don't they?

Reply:
I'm not sure to be honest. It is the same as https://w3c.github.io/remote-playback/#the-disableremoteplayback-attribute.
Which one sentence should I pick then?

Reply:
I think the second sentence is enough.